### PR TITLE
Fix image for add_contact not appearing in UG

### PR DESCRIPTION
**Pull request by tjch-o**

### CODE REVIEW

File: docs/UserGuide.md
Comment (rionshocker) [inline docs/UserGuide.md:1]: Don't change the table of contents link please

File: docs/UserGuide.md [hunk 1]
```diff
@@ -155,7 +159,7 @@ Examples:
 * `add_contact n/John Doe p/98765432 e/johndtr@example.com a/John street, block 123, #01-01`
```
Comment (rionshocker) [inline docs/UserGuide.md:159]: A spacing needed between `</box>` and the next line, otherwise, formatting gets truncated.

**Review by rionshocker — CHANGES_REQUESTED** [review]
Apart from some minor nits, LGTM!

File: docs/UserGuide.md
Comment (tjch-o) [reply docs/UserGuide.md:1]: that one was because local branch merge with master which is updated...

**Pull request closed without merging**